components/03-modules/footer/_footer.scss

```diff
@@ -69,6 +72,13 @@ $footer__bottom-bar-font-size        : $font-size-small !default;
                 border-bottom: 0;
             }
         }
+
+        .list__item {
+            padding-left: $footer__link-inner-padding;
+            @include mq($screen-m) {
+                padding-left: 0;
```

Comment:
Maybe it would be better to create new variable with modifier `--medium`

Reply:
yup, I've updated those styles